From: Federico Ceratto <federico@debian.org>
Date: Sun, 15 Dec 2013 01:41:56 +0000
Subject: Remove bottle.py from scripts

---
 setup.py | 1 -
 1 file changed, 1 deletion(-)

--- a/setup.py
+++ b/setup.py
@@ -17,7 +17,6 @@
       author_email='marc@gsites.de',
       url='http://bottlepy.org/',
       py_modules=['bottle'],
-      scripts=['bottle.py'],
       license='MIT',
       platforms = 'any',
       classifiers=['Development Status :: 4 - Beta',
